resources/resources/pe/itemmapping.json

```diff
@@ -159,6 +159,26 @@
     "peid": 16,
     "pedata": 0
   },
+  {
+    "itemkey": "oak_wood",
```

Comment:
these duplicate the log IDs, but the log mapping takes precedent for the reverse mapping. otherwise idk what to do with these...

Reply:
What do you mean?

Reply:
oak_log and the other variants use the same PE id and data =\ think i might just remove the wood for now, the mapping here is kinda useless anyways.

Reply:
Is this due to the new log vs stripped log types?

Reply:
yea =\ actually now that i think of it, theres probably a pedata value somewhere that works for those. hmmm

Reply:
If its they map to the same block, it should be put inside LegacyBlockData. We only map PC -> PE **definition** in here. If you need to remap a missing block it goes in legacyblockdata.

Reply:
yea, this is just for the new log item types, now that on 1.13 theres an actual item for the full bark log